tests/queries/0_stateless/02310_generate_multi_columns_with_uuid_v7.sql

```diff
@@ -0,0 +1,17 @@
+SELECT generateUUIDv7(1) = generateUUIDv7(2);
```

Comment:
Let's call this test + the expected result file just `02310_uuid_v7.sql/reference`.